### PR TITLE
API Webpack plus reloading via nodemon

### PR DESCRIPTION
**Pull request by nazar**
1. Added webpack3, configured for a node application. We should be able to use more es6 and even es7 language features plus using `import` and `export`. I've also setup folder aliases so we can do things like `import security from 'middleware/security'`, without specifying the `middleware` folderrelative path

2. added nodemon to monitor the `dist` folder, which is where webpack compiles the transpiled app to and restarts whenever the `dist` folder is changed.

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -6,9 +6,10 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node app.js"
+    "start": "webpack --watch & nodemon --watch dist dist/index.js & wait"
   },
   "dependencies": {
+    "babel-core": "^6.26.0",
```
Comment (mikelax) [inline api/package.json]: @nazar  this line and the one in devDependencies with the ^ is producing a warning. 

```
07:51 $ yarn install
yarn install v1.2.1
warning package.json: "dependencies" has dependency "babel-core" with range "^6.26.0" that collides with a dependency in "devDependencies" of the same name with version "6.26.0"
warning react-api-poc@0.1.0: "dependencies" has dependency "babel-core" with range "^6.26.0" that collides with a dependency in "devDependencies" of the same name with version "6.26.0"
```

**Review by mikelax — COMMENTED** [review]
@nazar  do you want to add a quick update to the readme to indicate the new lib and style used in the api project?

maybe add a couple of lines to [this section}(https://github.com/mikelax/react-api-poc#additional-libraries--integrations)?

You can also check off some of the TODO items with this PR.

**Comment by nazar** [comment]
Just pushed another update @mikelax

**Review by mikelax — APPROVED** [review]
on 🔥 
:shipit:

**Pull request merged**